app/assets/stylesheets/home.css

```diff
@@ -1,14 +1,171 @@
 @import url("./components/containers.css");
 @import url("./components/fancy_headers.css");
 
+@import url('https://fonts.googleapis.com/css2?family=Roboto&family=Sansita&display=swap');
+
 .container {
   display: flex;
   align-items: center;
-  justify-content: center;
 }
```

Comment:
Would you please remove the `.container` code here? This is already in the `.container` portion in `containers.css`.